Tests/DatadogTests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -26,15 +67,19 @@ class CrashReportingWithRUMIntegrationTests: XCTestCase {
             lastRUMViewEvent: activeRUMView // means there was a RUM session and it was sampled
         )
 
-        let integration: CrashReportingWithRUMIntegration = .mockWith(
-            core: core,
+        let receiver: CrashReportReceiver = .mockWith(
             dateProvider: RelativeDateProvider(using: currentDate),
             sessionSampler: Bool.random() ? .mockKeepAll() : .mockRejectAll(), // no matter sampling (as previous session was sampled)
             backgroundEventTrackingEnabled: .mockRandom() // no matter BET
         )
 
         // When
-        integration.send(report: crashReport, with: crashContext)
+        XCTAssertTrue(
+            receiver.write(crash: [
+                "report": crashReport,
+                "context": crashContext
+            ], to: core)
```

Comment:
Like I mentioned earlier, there is no reason to have more than one internal API available on `receiver` (the `.receive(message:from:)`) and keep both `write()` and `send()` private. WDYT?

Reply:
+1

Reply:
Indeed, I will change that 👍